src/main/java/org/jfrog/hudson/ArtifactoryServer.java

```diff
@@ -101,12 +101,12 @@ public ArtifactoryServer(String serverId, String artifactoryUrl, CredentialsConf
         this.deploymentThreads = deploymentThreads != null && deploymentThreads > 0 ? deploymentThreads : DEFAULT_DEPLOYMENT_THREADS_NUMBER;
     }
 
-    public String getName() {
+    public String getServerId() {
         return id;
     }
```

Comment:
FYI this change to a public Java API broke compatibility with my Groovy initialization script (which was not documented in the Release Notes) and required a code change.

Reply:
Thanks for letting us know @basil and apologies for not specifying this in the release notes!
We'll make sure to mark such changes as breaking changes as part of the release notes in the future.